jekyll/_cci2/runner-installation.adoc

```diff
@@ -33,9 +33,9 @@ Running jobs requires you have the following tools available on your machine:
 
 == Authentication
 
-NOTE: These commands can only be run by an owner/admin of your organization.
+NOTE: If you are installing **runner for server**, the CircleCI CLI needs to be configured using your server API key. Run `circleci setup` to configure the CLI and access the option to supply a new API token.
 
-In order to complete this process you will need to create a namespace and authentication token by performing the steps listed below:
+In order to install runner you will need to create a namespace and authentication token by performing the steps listed below – these commands can only be run by an owner/admin of your organization:
```

Comment:
I just moved this line down so we don't have too many NOTES at the start of the section.